### PR TITLE
Adding jecoute preview on chef mun overview

### PR DESCRIPTION
**Pull request by enzozccr**

### CODE REVIEW

File: templates/municipal_chief/home.html.twig [hunk 1]
```diff
@@ -86,7 +86,21 @@
...
+                    <h1 class="manager-overview__case--title">J'écoute</h1>
+                    <p class="manager-overview__case--baseline">Application disponible sur
+                        <a class="text--blue--dark link--no-decor" target="_blank" href="https://play.google.com/store/apps/details?id=fr.en_marche.jecoute&hl=it">Google Play</a>
+                        et sur l'<a class="text--blue--dark link--no-decor" target="_blank" href="https://apps.apple.com/fr/app/j%C3%A9coute/id1441973895">App Store</a>.
+                    </p>
+                    <footer class="manager-overview__case--footer">
+                        <a href="{{ path('app_jecoute_municipal_chief_local_surveys_list') }}" target="_blank" class="btn btn btn--ghosting--blue">Administrer</a>
+                    </footer>
+                </div>
+                {% else %}
```
Comment (ottaviano) [inline templates/municipal_chief/home.html.twig]: could you remove the `else` statement? 
```twig
{% if ... %}
   ....
{% else %}
    </div></div>
{% endif %}
```
=>
```twig
{% if ... %}
   ....
{% endif %}
```

File: templates/municipal_chief/home.html.twig [hunk 2]
```diff
@@ -86,7 +86,21 @@
                     <a href="https://chezvous.en-marche.fr/" target="_blank" class="btn btn btn--ghosting--blue">Consulter</a>
                 </footer>
             </div>
-            <div></div>
+            {% if app.user.municipalChiefManagedArea.hasJecouteAccess() %}
+                <div class="manager-overview__case">
+                    <span class="manager-overview__case--section">Communication & Mobilisation</span>
+                    <h1 class="manager-overview__case--title">J'écoute</h1>
+                    <p class="manager-overview__case--baseline">Application disponible sur
+                        <a class="text--blue--dark link--no-decor" target="_blank" href="https://play.google.com/store/apps/details?id=fr.en_marche.jecoute&hl=it">Google Play</a>
```
Comment (ottaviano) [inline templates/municipal_chief/home.html.twig]: `...play.google.com/....&hl=it` 
`hl=it` it's for the Italian version of the page 🇮🇹 🍝

File: templates/municipal_chief/home.html.twig (hunk 1 above)
Comment (ottaviano) [reply templates/municipal_chief/home.html.twig]: ohh I noticed that you replace a div element `<div></div>` by two closing tags `</div></div>`. if you need to keep `else` statement maybe you should change two closing tags by `<div></div>`?

image
image

Comment (enzozccr) [reply templates/municipal_chief/home.html.twig]: I need to have at least 2 elements in a row to have it at 50% width, I use an empty markeup when I only have one, could I do either ?

Comment (enzozccr) [reply templates/municipal_chief/home.html.twig]: > ohh I noticed that you replace a div element `<div></div>` by two closing tags `</div></div>`. if you need to keep `else` statement maybe you should change two closing tags by `<div></div>`?
> 
> image
> image

ah you're right there is a `/` too much !

File: templates/municipal_chief/home.html.twig (hunk 2 above)
Comment (enzozccr) [reply templates/municipal_chief/home.html.twig]: hahaha correct

**Comment by enzozccr** [comment]
everything should be fixed!

**Pull request merged**